### DISCUSSION TITLE
Make an organization Readme

### DISCUSSION THREAD

**Discussion by tmigot** [JuliaSmoothOptimizers/Organization#30] [Ideas]
The idea is to make a README for the organization. This is suggested on the right-hand side of the main organization page
https://github.com/JuliaSmoothOptimizers/
and here is further doc
https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile

We could give some "guidelines" or the "spirit" of the organization and point toward JuliaCon's presentation and the website.

**Comment by dpo** [comment]
I started working on one a while back. I'll try and push it soon.
